model-optimizer/extensions/ops/non_max_suppression.py

```diff
@@ -47,7 +46,7 @@ def __init__(self, graph: Graph, attrs: dict):
 
     def backend_attrs(self):
         version = self.get_opset()
-        if version in ['opset3', 'opset4']:
+        if version in ['opset3', 'opset4', 'opset5']:
```

Comment:
Please, make sure that the IR containing NMS-4 can be read and written by MO IR reader correctly.

Reply:
Done.